Mathlib/MeasureTheory/Integral/RieszMarkovKakutani.lean

```diff
@@ -245,9 +257,142 @@ lemma rieszContentAux_union {K₁ K₂ : TopologicalSpace.Compacts X}
+          max_eq_left <| le_of_lt (lt_of_le_of_lt zero_le_one hα)]
+        exact mul_comm _ _
+      rw [this]
+      set K' := f ⁻¹' Ici α⁻¹.toNNReal with hK'
+      have hKK' : K.carrier ⊆ interior K' := by
+        rw [subset_interior_iff]
+        use f ⁻¹' (Ioi α⁻¹.toNNReal)
+        refine ⟨IsOpen.preimage f.1.2 isOpen_Ioi, ?_, ?_⟩
+        · intro x hx
+          rw [Set.mem_preimage, Set.mem_Ioi]
+          exact lt_of_lt_of_le (Real.toNNReal_lt_one.mpr (inv_lt_one_of_one_lt₀ hα))
+            (hfleoneonK x hx)
+        · rw [hK']
+          intro x hx
+          simp only [mem_preimage, mem_Ioi] at hx
+          simp only [mem_preimage, mem_Ici]
+          exact le_of_lt hx
+      have hK'cp : IsCompact K' := by
+        apply IsCompact.of_isClosed_subset f.2
+        · exact IsClosed.preimage f.1.2 isClosed_Ici
+        · rw [hK']
+          apply Set.Subset.trans _ subset_closure
+          intro x hx
+          simp only [mem_preimage, mem_Ici] at hx
+          simp only [mem_support]
+          apply ne_of_gt
+          rw [Real.toNNReal_inv] at hx
+          exact (lt_of_lt_of_le
+            (inv_pos_of_pos (lt_trans zero_lt_one (Real.one_lt_toNNReal.mpr hα))) hx)
+      set hb' := hb ⟨K', hK'cp⟩
+      simp only [Compacts.coe_mk, le_iInf_iff] at hb'
+      have hbK' : b ≤ rieszContent Λ ⟨K', hK'cp⟩ := hb' hKK'
+      rw [ENNReal.le_coe_iff] at hbK'
+      obtain ⟨p, hp⟩ := hbK'
+      simp only [hp.1, ENNReal.toNNReal_coe, NNReal.val_eq_coe, map_smul, smul_eq_mul,
+        NNReal.coe_mul, Real.coe_toNNReal', ge_iff_le]
+      apply le_trans (NNReal.GCongr.toReal_le_toReal hp.2)
+      simp only [rieszContent]
+      rw [rieszContentAux, ← Real.coe_toNNReal (α ⊔ 0) (le_max_right α 0), ← NNReal.coe_mul,
+        NNReal.coe_le_coe]
+      apply csInf_le
+      · simp only [OrderBot.bddBelow]
+      · simp only [mem_image, mem_setOf_eq]
+        use α.toNNReal • f
+        refine ⟨?_, ?_⟩
+        · intro x hx
+          simp only [CompactlySupportedContinuousMap.coe_smul, Pi.smul_apply, smul_eq_mul,
+            ← NNReal.coe_le_coe, NNReal.coe_one, NNReal.coe_mul, Real.coe_toNNReal']
+          rw [← (left_eq_sup.mpr <| le_of_lt (lt_of_le_of_lt zero_le_one hα)), mul_comm]
+          apply (inv_le_iff_one_le_mul₀ (lt_trans zero_lt_one hα)).mp
+          rw [← Set.mem_Ici]
+          simp only [mem_Ici, ge_iff_le]
+          exact Real.toNNReal_le_iff_le_coe.mp hx
+        · simp only [map_smul, smul_eq_mul, mul_eq_mul_right_iff]
+          left
+          rw [Real.toNNReal_eq_toNNReal_iff (le_of_lt (lt_of_le_of_lt zero_le_one hα))
+            (le_max_right α 0), left_eq_sup]
+          exact le_of_lt (lt_of_le_of_lt zero_le_one hα)
+    · exact zero_le (Λ f)
+
+end RieszContentRegular
+
+section RieszMeasure
+
+variable [T2Space X] [LocallyCompactSpace X] [MeasurableSpace X] [BorelSpace X]
+
+/-- `rieszContent` is promoted to a measure. -/
+def rieszMeasure := (rieszContent Λ).measure
```

Comment:
Can you make the docstring not rely on context that the reader can only access by scrolling through the file?

Currently if I hover over `rieszMeasure` and see
```
`rieszContent` is promoted to a measure.
```
I still don't know what `rieszMeasure` is